onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -196,10 +196,10 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level,
       ORT_ENFORCE(fetches.size() == node->OutputDefs().size());
       converted_to_constant = true;
       for (size_t fetch_idx = 0; fetch_idx < fetches.size(); ++fetch_idx) {
-        OrtValue& ort_value = fetches[fetch_idx];
+        auto* constant_arg_out = node->MutableOutputDefs()[fetch_idx];
         // XXX: Add support for SparseTensors outputs when we have sparse outputs
```

Comment:
do we need a mutable NodeArg instance here?

nit: would be slightly nicer to use NodeArg& to make it clear you don't expect a nullptr to be returned by the call to get the NodeArg.

Reply:
Done